examples/etcd/etcd-in-cluster/ocp/README.md

```diff
@@ -0,0 +1,274 @@
+on the restore host, below steps can be followed to restore ETCD:
+
+- Download the ETCD snapshot on the restore host using the aws cli on a specific path let's say `/var/home/core/etcd-backup`
+- Stop the static pods from all other leader hosts (not the recovery host) by copying the manifests out of the static pod path dir, i.e. `/etc/kubernetes/manifests`
+
+  ```
+  # move etcd pod manifest
+  sudo mv /etc/kubernetes/manifests/etcd-pod.yaml /tmp
+
+  # make sure etcd pod has been stopped
+  sudo crictl ps | grep etcd
+
+  # move api server pod
+  sudo mv /etc/kubernetes/manifests/kube-apiserver-pod.yaml /tmp
+  ```
+
+- move the etcd data dir to a different location
+  ```
+  sudo mv /var/lib/etcd/ /tmp
+  ```
+
+  Repeast these steps on all other leader hosts that is not the restore host.
+
+- Run the `cluster-ocp-restore.sh` script with the location where you have downloaded the etcd snapshot that in our case is `/var/home/core/etcd-backup`
+
+  ```
+  sudo ./cluster-ocp-restore.sh /var/home/core/etcd-backup
+  ```
+
+- Restart `kubelet` service on all the leader nodes
+
+  ```
+  sudo systemctl restart kubelet.service
+  ```
+
+- Verify single ETCD node has been started, run below from recovery host to check if ETCD container is up
+
+  ```
+  sudo crictl ps | grep etcd
+
+  # you can also verify that the ETCD pod is running now.
+  root@workmachine:/repo# oc get pods -n openshift-etcd
+  NAME                                                           READY     STATUS      RESTARTS   AGE
+  etcd-ip-10-0-149-197.us-west-1.compute.internal                1/1       Running     0          3m57s
+  installer-2-ip-10-0-149-197.us-west-1.compute.internal         0/1       Completed   0          7h54m
+  installer-2-ip-10-0-166-99.us-west-1.compute.internal          0/1       Completed   0          7h53m
+  installer-2-ip-10-0-212-253.us-west-1.compute.internal         0/1       Completed   0          7h52m
+  revision-pruner-2-ip-10-0-149-197.us-west-1.compute.internal   0/1       Completed   0          7h51m
+  revision-pruner-2-ip-10-0-166-99.us-west-1.compute.internal    0/1       Completed   0          7h51m
+  revision-pruner-2-ip-10-0-212-253.us-west-1.compute.internal   0/1       Completed   0          7h51m
+
+  ```
+
+- Force ETCD deployment, you can run below command from the terminal you have cluster access
+  ```
+  oc patch etcd cluster -p='{"spec": {"forceRedeploymentReason": "recovery-'"$( date --rfc-3339=ns )"'"}}' --type=merge
+
+  # Verify all nodes are updated to latest version
+  oc get etcd -o=jsonpath='{range .items[0].status.conditions[?(@.type=="NodeInstallerProgressing")]}{.reason}{"\n"}{.message}{"\n"}'
+  ```
+
+  And you will get message like
+
+  ```
+  3 nodes are at revision 2; 0 nodes have achieved new revision 3
+  ```
+  Please wait for some time make sure the component has been updated to the latest version, and then the message would look somewhat like this
```

Comment:
```suggestion
  Please wait for some time make sure the component has been updated to the latest version, and then the message would look like this
```

Reply:
the command is not actually gonna look exactly like what is mentioned, thats why I said somewhat.

Reply:
resolving